.github/workflows/release.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+      - run: npm install
+      - run: npm run check
```

Comment:
It's better to have this run as a dependency on the existing test steps. The Sass repo does this today, and you can follow that pattern (defining the release Yaml as `on: {workflow_call:}` and having the test Yaml invoke it using `uses:`).